TASVideos/WikiModules/BannedUsers.cshtml.cs

```diff
@@ -0,0 +1,25 @@
+﻿using TASVideos.WikiEngine;
+
+namespace TASVideos.WikiModules;
+
+[WikiModule(ModuleNames.BannedUsers)]
+public class BannedUsers(ApplicationDbContext db) : WikiViewComponent
+{
+	public PageOf<Entry> Users { get; set; } = new([], new());
+
+	public async Task<IViewComponentResult> InvokeAsync()
+	{
+		Users = await db.Users
+			.ThatAreBanned()
+			.OrderBy(u => u.UserName)
+			.Select(u => new Entry(
+				u.UserName,
+				u.BannedUntil!.Value,
+				u.LastLoggedInTimeStamp,
+				u.ModeratorComments))
+			.SortedPageOf(GetPaging());
+		return View();
+	}
+
+	public record Entry(string Name, DateTime BannedUntil, DateTime? LastLoggedIn, string? ModeratorComments);
```

Comment:
to get sorting, you have to put the sortable attribute on the properties of this Entry type.  Which means it probably will need to be a class with init properties instead of record

Reply:
I'm only getting "The LINQ expression ... could not be translated."

Reply:
did you make it a class? Records can't be used with the sorting and paging, for some spooky Entity Framework reasons